docs/dml-of-carbondata.md

```diff
@@ -352,18 +352,18 @@ CarbonData DML statements are documented here,which includes:
   ```
   LOAD DATA [LOCAL] INPATH 'folder_path' 
   INTO TABLE [db_name.]table_name PARTITION (partition_spec) 
-  OPTIONS(property_name=property_value, ...)    
+  OPTIONS(property_name=property_value, ...);    
```

Comment:
Better to add a blank line between these two sql.

Reply:
The seperator is used in beeline , but not in `spark.sql(...)`.
Use blank line to seperate OR use different code block of Markdown instead.